scripts/ci-skip-native-if-possible.js

```diff
@@ -29,7 +29,7 @@ if (branch === "master") {
 }
 
 /** @type {String[]} */
-const changedFiles = reverseSh(`git diff --name-only HEAD $(git merge-base HEAD origin/master)`).split("\n")
+const changedFiles = reverseSh(`git diff --name-only HEAD origin/master`).split("\n")
```

Comment:
I'm curious about this change? Was it leading to false positives?

Reply:
Made this change with @sweir27 during future fursday but forgot why now.

Will undo it for now and if I run into issues while deploying this change I'll look into adding it back.